test/test.spec.js

```diff
@@ -70,16 +71,12 @@ describe('The CLI', function () {
 });
 
 describe('The server', function () {
-  var server;
-  before('Start running the server', function() {
+  let server;
+  before('Start running the server', function () {
```

Comment:
They're [discouraged in Mocha](https://mochajs.org/#arrow-functions) because they break the calling context, which carries some of Mocha's interface. I wrote an [eslint rule](https://github.com/lo1tuma/eslint-plugin-mocha/pull/78) we could use.